apps/progs.pl

```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/env perl
-# Copyright 1995-2016 The OpenSSL Project Authors. All Rights Reserved.
+# Copyright 1995-2017 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Whoa.  This is a meta-copyright change :)  Changing the copyright year on the generated code means changing the copyright year on the script doing the generating.  Neat.  Leave this.